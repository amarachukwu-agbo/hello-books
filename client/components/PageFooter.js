import React from 'react';

const PageFooter = () => (
    <footer className="page-footer foot lighten-5">
        <div className="footer-copyright">
            <div className="container center-align grey-text">
                Â© 2017 Copyright Hello-books
                    </div>
        </div>
    </footer>
);

export default PageFooter;
