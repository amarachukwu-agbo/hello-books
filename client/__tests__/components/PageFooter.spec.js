import React from 'react';
import renderer from 'react-test-renderer';
import PageFooter from '../../components/Common/PageFooter';

describe('<PageFooter>', () => {
  it('should render the footer correctly', () => {
    const tree = renderer.create(<PageFooter />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
