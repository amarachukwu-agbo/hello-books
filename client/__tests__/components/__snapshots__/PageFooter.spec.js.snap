// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageFooter> should render the footer correctly 1`] = `
<footer
  className="page-footer foot lighten-5"
>
  <div
    className="footer-copyright"
  >
    <div
      className="container center-align grey-text"
    >
      Â© 2017 Copyright Hello-books
    </div>
  </div>
</footer>
`;
